lib/output.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+let cli = require('heroku-cli-util')
+
+let Stream = function (url) {
```

Comment:
I don't know. This is already copied from [heroku-apps](https://github.com/heroku/heroku-apps/blob/master/lib/output.js).
See [this comment](https://github.com/heroku/heroku-releases-retry/pull/2#discussion_r117073339).

I'd rather keep this as simple as possible, and if we do need explicit tests because it becomes more complex in the future, we should extract it and stop copying it across projects.